### PR TITLE
Improve normalization

### PR DESCRIPTION
**Pull request by SergeyTsimfer**

### CODE REVIEW

File: batchflow/models/torch/layers/normalization.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        self.eps = eps
+
+
+    def forward(self, x):
+        if self.data_format == "channels_last":
+            return F.layer_norm(x, (self.num_features,), self.weight, self.bias, self.eps)
+
+        # Use multiplication instead of `pow` as `torch2trt` throws warnings about it
+        mean = x.mean(1, keepdim=True)
+        normalized = x - mean
```
Comment (HollowPrincess) [inline batchflow/models/torch/layers/normalization.py:28]: Maybe `normalized` is not a suitable name for this variable?

File: batchflow/tests/notebooks/torch_benchmark.ipynb [hunk 2]
```diff
@@ -0,0 +1,4784 @@
+{
```
Comment (HollowPrincess) [inline batchflow/tests/notebooks/torch_benchmark.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #15.    from batchflow.models.torch import *
</pre><p>What's about <code>set_gpu()</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (HollowPrincess) [inline batchflow/tests/notebooks/torch_benchmark.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #16.            'K_1357': Block(inputs=inputs, layout='K', channels='same', kernel_size=(1, 3, 5, 7), dilation=1),
</pre><p>Not clear naming</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by HollowPrincess** [comment]
I think, the notebook needs some prettifyings in module namings. The wordsstucktogether are hard to read.

File: batchflow/tests/notebooks/torch_benchmark.ipynb (hunk 2 above)
Comment (AnnaAltynova) [inline batchflow/tests/notebooks/torch_benchmark.ipynb:1]: <p>adding more summarizing cells like this one after each dataframe would be nice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (AnnaAltynova) [inline batchflow/tests/notebooks/torch_benchmark.ipynb:1]: <p><code>&nbsp;# Amp speeds up things, but does not change memory required!</code></p><p><code>dataframe.groupby('amp').mean()</code></p><p>adding more summarizing cells like this one after each dataframe would be nice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Pull request merged**